docs/docs/spec/intro.md

```diff
@@ -3,7 +3,7 @@ title: "Introduction"
 sidebar_position: 1
 ---
  
-This is a specification for running of [EigenLayer](https://www.eigenlayer.xyz/) Nodes. AVS is used to participate in the EigenLayer restaking system, to know more about EigenLayer see its [documentation page](https://docs.eigenlayer.xyz/overview/readme). 
+This is a specification for running of [EigenLayer](https://www.eigenlayer.xyz/) AVS Nodes. AVS is used to participate in the EigenLayer restaking system, to know more about EigenLayer see its [documentation page](https://docs.eigenlayer.xyz/overview/readme). 
```

Comment:
remove "of" (just `running Eigenlayer AVS Nodes`
or find better way to say this

Reply:
I think `running Eigenlayer AVS Nodes` is fine.